Dockerfile.erb

```diff
@@ -37,6 +27,10 @@ RUN set -ex \
 	&& tar -xzvf $BITCOIN_DIST -C /usr/local --strip-components=1 --exclude=*-qt \
 	&& rm bitcoin*
 
+RUN apt-get autoremove -qqy --purge ca-certificates wget gpg dirmngr \
+	&& apt-get clean \
+	&& rm -rf /var/lib/apt/lists/*
```

Comment:
this step is not achieving anything, due to the way docker layers work you will not reduce the total size of container (and likely even increase it slightly)